docs/Tutorials/Kubernetes/Create-Cluster.md

```diff
@@ -0,0 +1,232 @@
+to deploy Hyperledger Besu.
+
+To create a cluster in AWS, you need to install the [AWS CLI](https://aws.amazon.com/cli/) and
+[`eksctl`](https://eksctl.io/).
+
+The [template](https://github.com/ConsenSys/quorum-kubernetes/tree/master/aws) comprises the base
+infrastructure used to build the cluster and other resources in AWS. We also use AWS native
+services and features after the cluster is created. These include:
+
+* [Pod identities](hhttps://github.com/aws/amazon-eks-pod-identity-webhook)
+* [Secrets Store CSI drivers](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html)
+* Dynamic storage classes backed by AWS EBS. The
+    [volume claims](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) are fixed
+    sizes and can be updated as you grow via helm updates, and will not need to re-provision the underlying storage
+    class.
+* [CNI](https://docs.aws.amazon.com/eks/latest/userguide/pod-networking.html) networking mode for EKS. By default,
+    EKS clusters use `kubenet` to create a virtual network and subnet. Nodes get an IP
+    address from a virtual network subnet. Network address translation (NAT) is then configured on the nodes, and pods
+    receive an IP address "hidden" behind the node IP.
+
+    !!! note
+        This approach reduces the number of IP addresses that you need
+        to reserve in your network space for pods, but places constraints on what can connect to the nodes from
+        outside the cluster (for example on premises nodes or those on another cloud provider).
+
+AWS Container Networking Interface (CNI) provides each pod with an IP address from the subnet, and can be accessed
+directly. The IP addresses must be unique across your network space, and must be planned in advance. Each node has
+a configuration parameter for the maximum number of pods that it supports. The equivalent number of IP addresses
+per node are then reserved up front for that node. This approach requires more planning, and can lead to IP address
+exhaustion as your application demands grow, however makes it easier for external nodes to connect to your cluster.
+
+!!!warning
+
+    EKS clusters may not use 169.254.0.0/16, 172.30.0.0/16, 172.31.0.0/16, or 192.0.2.0/24 for the Kubernetes
+    service address range.
+
+To provision the cluster:
+
+1. Update [cluster.yml](https://github.com/ConsenSys/quorum-kubernetes/blob/master/aws/templates/cluster.yml) with
+your VPC details.
+
+1. Deploy the template:
+
+    ```bash
+    eksctl create cluster -f ./templates/cluster.yml
+    ```
+
+1. Optionally, deploy the
+[kubernetes dashboard](https://github.com/ConsenSys/quorum-kubernetes/tree/master/aws/templates/k8s-dashboard)
+
+1. Provision the drivers. After the deployment completes, provision the secrets manager, identity, and
+CSI drivers. Use `besu` or `quorum` for `EKS_NAMESPACE` and update `AWS_REGION` and `EKS_CLUSTER_NAME` in the
```

Comment:
@joshuafernandes, where possible could we remove references to quorum?

Reply:
Yep, missed this one. I had updated everything to say `besu`

Reply:
Fixed mate :)